scootapi/client/get_status_cmd.go

```diff
@@ -19,6 +20,11 @@ func (c *getStatusCmd) registerFlags() *cobra.Command {
 func (c *getStatusCmd) run(cl *Client, cmd *cobra.Command, args []string) error {
 
 	log.Println("Checking Status for Scoot Job", args)
+
+	if args == nil || len(args) == 0 {
```

Comment:
I think len(args) returns 0 if args == nil, so no need for a separate check (unlike java)